libs/cart/state/testing/src/factories/stateful-composite-cart-item.factory.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+
+import { DaffModelFactory } from '@daffodil/core/testing';
+import { DaffMockCompositeCartItem } from '@daffodil/cart/testing';
+import { DaffStatefulCartItem, DaffCartItemStateEnum, DaffStatefulCompositeCartItem } from '@daffodil/cart/state';
+
+export class DaffMockStatefulCompositeCartItem extends DaffMockCompositeCartItem implements DaffStatefulCartItem {
```

Comment:
Should implement `DaffStatefulCompositeCartItem`.

Reply:
fixed